src/applications/vaos/components/calendar/CalendarCell.jsx

```diff
@@ -25,21 +25,22 @@ const CalendarCell = ({
   });
 
   return (
-    <button
-      id={`date-cell-${date}`}
-      className={cssClasses}
-      onClick={() => onClick(date)}
-      disabled={disabled}
-      aria-label={ariaDate}
-      aria-expanded={isCurrentlySelected}
-      type="button"
-    >
-      {inSelectedArray && <i className="fas fa-check vads-u-color--white" />}
-      {dateDay}
-      {isCurrentlySelected && (
-        <span className="vaos-calendar__cell-selected-triangle" />
-      )}
-    </button>
+    <div role="cell" className={cssClasses}>
```

Comment:
This works exactly as expected. Can we wrap the button on line 14 in a `div[role="cell"] also? Counts get off in VoiceOver, and likely other screen readers) when each cell isn't accounted for.

```javascript
...
return (
  <div role="cell" className={cssClasses}>
    <button className="vaos-calendar__calendar-day vads-u-visibility--hidden" />
  </div>
);
```